docs/source/api/methods.rst

```diff
@@ -1,50 +1,60 @@
-****************
+*****************
```

Comment:
Is there a reason you changed all the rst headlines markings? If you change them, make sure the number of markers is the same as the number of characters in the headline

Reply:
I restructured them a bit differently. This is how it renderes:
<img width="299" alt="Screenshot 2024-06-11 at 19 08 21" src="https://github.com/PedestrianDynamics/PedPy/assets/5772973/8e774b50-c3b8-4ffb-b2a5-3d76d8d21aea">